_data/tool_and_resource_list.yml

```diff
@@ -2279,3 +2279,13 @@
   url: https://aria.services
   registry:
     biotools: aria_access
+- description: Data Management Plan (DMP) generator that focuses on plant science.
+  id: dataplan
+  name: DataPLAN
+  url: https://plan.nfdi4plants.org
+  registry:
+    biotools: dataplan
+- description: Online questionnaire for the development of data management plans, and DMP repository.
+  id: opidor
```

Comment:
I just noticed that OPIDoR is already listed under the National resources and is now being duplicated. We will also lose the info that it is an instance of dmproadmap if we take it out of the national resources. Maybe we leave it out @erlefloch ?

Reply:
Sure ! Let's do so

Reply:
done in https://github.com/erlefloch/rdmkit/commit/6908f58ce731ba8735c44befd5189a640cc47ff6